azure-pipelines.yml

```diff
@@ -1,19 +1,42 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
-
 trigger:
 - master
 
-pool:
-  vmImage: 'ubuntu-latest'
+pool: UKHO Ubuntu 1804
```

Comment:
Should this be in the format:
pool:
    vmIMage: "UKHO Ubuntu 1804" ?

It might not : )

Reply:
No, that's in the correct for this :)